cmake/Modules/BuildCommon.cmake

```diff
@@ -44,6 +44,7 @@ set(BUILD_SHARED_LIBS YES CACHE BOOL "Build shared libraries. Default: YES")
 # that this is a standard option.
 set(BUILD_TESTING ON CACHE BOOL "Build tests")
 set(BUILD_DOCS OFF CACHE BOOL "Build documentation with make doc")
+set(BUILD_EXTERNAL_DOCS OFF CACHE BOOL "Hide '*-draft*' documentation pages and directories when building documentation")
```

Comment:
- Preference for making the default behavior to build external docs (i.e., not build internal ones).

- Preference for a single option `BUILD_DOCS`; truthy values or `external` build the external docs; falsey values don't build any docs; `BUILD_DOCS=internal` builds the internal docs. (CMake `if` should handle truthy and falsey automatically). If you do change the meaning of `BUILD_DOCS`, grep around for previous uses of `BUILD_DOCS` and replace them with `BUILD_DOCS=external`

Reply:
I can do that. Do we want to default to building docs? This was not the original behavior

Reply:
>> Preference for making the default behavior to build external docs (i.e., not build internal ones).

> I can do that. Do we want to default to building docs? This was not the original behavior

Sorry, clarification:

```
set(BUILD_DOCS OFF CACHE BOOL "Build documentation with make doc")
```

as before but where previously `BUILD_DOCS` could be truthy or falsy, instead: falsy, `internal` or other truthy. I.e., treat truthy as `external`